### ISSUE TITLE
msgflo-foreign-participant MQTT timeout

### ISSUE THREAD

**Issue by bergie** [msgflo/msgflo#62]
After a while of running the coordinator, seeing one of these per each foreign participant (started with `--forever=true`):

```
1485432445: Client mqttjs_aa005680 has exceeded timeout, disconnecting.
1485432445: Socket error on client mqttjs_aa005680, disconnecting.
```

#30 would probably help?

**Comment by jonnor** [comment]
Yes, we must send regularly to not be disconnected on MQTT. One can bump keepalive, but its only delaying the inevitable.

**Comment by jonnor** [comment]
Considering we don't actually send anything after the initial message this doesn't really result in any observable issues, apart from that covered in #30

**Comment by jonnor** [comment]
@bergie this should be fixed now in `0.10.0`. Can you re-rest?

**Comment by jonnor** [comment]
Please re-open if not fixed.
